nxdrive/options.py

```diff
@@ -370,39 +369,6 @@ def server_updater(*args: Any) -> "ServerOptionsUpdater":
     We use this to prevent loading any Drive related stuff and keep
     the possibility to import other classes without anything else needed.
     """
-
```

Comment:
The whole `server_updater()` function should be deleted from the file. It can be completely removed and the `ServerOptionsUpdater` class must be used directly when needed.
(+ a line in the `technical_changes.md`)